### DISCUSSION TITLE
IEA-15-240-RWT-UMaineSemi platform initial value is not zero

### DISCUSSION THREAD

**Discussion by kuanyuuu-chen** [OpenFAST/openfast#1216] [Q&A]
Hello,

I'm trying to run D:\ROSCO-main\Test_Cases\IEA-15-240-RWT-UMaineSemi with openfast.
but I got nonzero platform pitch and surge. I've tried to fixed it by changing the setting in .fst file to let it start from zero but failed.

![pic1](https://user-images.githubusercontent.com/81912241/184672343-acffd9ea-f9f7-436a-a230-a401c17ddc5a.png)

![pic](https://user-images.githubusercontent.com/81912241/184672351-2557f029-a46a-4297-a302-1c4119253e7d.png)

Can anyone tell me which parameter and which file should I edit?
Best regards,

**Comment by jjonkman** [comment]
Dear @kuanyuuu-chen,

The initial conditions of the ElastoDyn DOFs are defined in the ElastoDyn input file (in this example, _Test_Cases/IEA-15-240-RWT-UMaineSemi/IEA-15-240-RWT-UMaineSemi_ElastoDyn.dat_).  You'll see in this file, unless you've changed it, that `PtfmSurge` = 5 m and `PtfmPitch` = 3 deg.

Best regards,

**Reply by kuanyuuu-chen** [reply]
thanks a lot!!!
it works!!!!

**Reply by unistudent2018** [reply]
Hello kuanyuuu-chen,

I am trying to run the 15MW UMaineSemi sub but I keep getting errors when trying to run it. Are you able to help?
